java-checks/src/test/files/checks/CollectionInappropriateCallsCheck.java

```diff
@@ -0,0 +1,114 @@
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+class A {
```

Comment:
Lacks the case of List<Number> l = Lists.<Integer>newArrayList(1);  l.contains(1);